packages/client/package.json

```diff
@@ -3,6 +3,10 @@
   "version": "0.1.2",
   "main": "dist/main.js",
   "module": "src/index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/wallix/webauthn/tree/master/packages/client"
```

Comment:
Please use the main repo URL and add a `directory` field instead of linking to a subtree. Same for the other package.

See https://docs.npmjs.com/files/package.json#repository